packages/apps/graph/generated-schema.graphql

```diff
@@ -3,8 +3,11 @@ The `BigInt` scalar type represents non-fractional signed whole numeric values.
 """
 scalar BigInt
 
+"""A unit of information that stores a set of verified transactions."""
```

Comment:
Would this also be true for an orphan block

Reply:
Something to review when we implement orphans I think